packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -392,7 +392,7 @@ function* watchFetchChats() {
 }
 
 function* watchFetchChatMessages() {
-  yield takeLatest(fetchMoreMessages, doFetchMoreMessages)
+  yield takeEvery(fetchMoreMessages, doFetchMoreMessages)
```

Comment:
would you mind also doing a pass of the `doFetchMoreMessages` saga and making sure we take the proper max/min of the pointers?

Reply:
sorry not sure what you mean by this? logic seems ok to me.

Reply:
https://github.com/AudiusProject/audius-client/blob/72fa511ef95cba111f67976c8ab24b6dfb9c12cf/packages/common/src/store/pages/chat/slice.ts#L194

We take the summary from the latest fetch, because we assume that's going to have the latest pointer/cursor, but we should probably be getting the min and max so the next fetch starts at the appropriate place regardless of order